datafusion-federation/Cargo.lock

```diff
@@ -1,2405 +0,0 @@
-# This file is automatically @generated by Cargo.
```

Comment:
Note this lock file was unused as it wasn't the toplevel workspace file. Should I commit the toplevel lock file?

Reply:
Hey @nathanielc, yes we should probably delete it and use the top-level Cargo.lock file.

Reply:
Looks like the toplevel Cargo.lock file has been explicitly ignored in the repo, as this repo only produces library crates and no binary crates that seems like a good decision. I'll leave the lock file out.